example_test.go

```diff
@@ -166,7 +166,7 @@ func ExamplePingRequest_Context() {
 	fmt.Println("Ping Error", err)
 	// Output:
 	// Ping Resp data []
-	// Ping Error context is done
+	// Ping Error context is done (requestId 3)
```

Comment:
The test may be unstable in the future, I suggest to cut the id part somehow:
```suggestion
	// Ping Error context is done
```